pallets/gear/src/benchmarking/syscalls.rs

```diff
@@ -156,10 +156,9 @@ where
         )
     }
 
-    pub fn alloc(r: u32) -> Result<Exec<T>, &'static str> {
+    pub fn alloc(repetitions: u32, pages: u32) -> Result<Exec<T>, &'static str> {
```

Comment:
what about assert here?
assert!(repetitions * pages * API_BENCHMARK_BATCH_SIZE <= MAX_PAGES)

Reply:
As I can see alloc_per_page does not fit this restriction.

Fix also:
```
Instruction::I32Const(i32::MAX),
```

May be we have now another constant